ecosystem/sep-direct-payment.md

```diff
@@ -0,0 +1,318 @@
+1. Once the stellar transaction is completed, the receiving anchor should deposit the money in the receivers bank account.
+1. The sending anchor can query the status of this transaction via the [`/transaction`](#transaction) endpoint, and should communicate updates to the sending client as that progresses.
+
+
+
+## API Endpoints
+
+* [`GET /info`](#info): required
+* [`POST /send`](#send): required
+* [`GET /transactions`](#transaction-history): required
+* [`GET /transaction`](#single-historical-transaction): required
+
+### Info
+#### Request
+
+```
+GET DIRECT_PAYMENT_SERVER/info
+```
+
+Allows an anchor to communicate basic info about what currencies their `DIRECT_PAYMENT_SERVER` supports receiving from partner anchors.
+
+Request parameters:
+
+Name | Type | Description
+-----|------|------------
+`lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `description` fields in the response should be in this language.
+
+#### Response
+
+The response should be a JSON object like:
+
+```json
+{
+   "receive":{
+      "USD":{
+         "enabled":true,
+         "fee_fixed":5,
+         "fee_percent":1,
+         "min_amount":0.1,
+         "max_amount":1000,
+         "fields":{
+            "sender":{
+               "first_name":"The sender's first name",
+               "last_name":"The sender's last name"
+            },
+            "receiver":{
+               "first_name":"The receiver's first name",
+               "last_name":"The receiver's last name",
+               "email_address":"The receiver's email address"
+            },
+            "transaction":{
+               "routing_number":{
+                  "description":"routing number of the destination bank account"
+               },
+               "account_number":{
+                  "description":"bank account number of the destination"
+               },
+               "type":{
+                  "description":"type of deposit to make",
+                  "choices":[
+                     "SEPA",
+                     "SWIFT"
+                  ]
+               }
+            }
+         }
+      }
+   }
+}
+```
+
+The JSON object contains an entry for each asset that the anchor supports for receiving and completing a direct payment.
+
+#### For each asset available for receiving, response contains:
+
+* `min_amount`: Optional minimum amount. No limit if not specified.
+* `max_amount`: Optional maximum amount. No limit if not specified.
+* `fee_fixed`: Optional fixed (flat) fee for deposit. In units of the received asset. Leave blank if there is no fee or the fee schedule is complex.
+* `fee_percent`: Optional percentage fee for deposit. In percentage points. Leave blank if there is no fee or the fee schedule is complex.
```

Comment:
If the fee schedule is complex, how is it communicated?

Reply:
I think we can not worry about it, and just find out the final fee once the transaction is created (but before any money is sent).  This was more important for interactive flow stuff because it makes a nicer UX to see it up front but i think in this case, especially since both ends might take a cut, there's no need to add a /fee endpoint back yet. 

Honestly we might even remove these fee values.